### PR TITLE
Add conda-forge and pypi badges + update installation instructions

### LINKED ISSUE #6: Set up scripts as a conda package
This is one suggestion for usability. The notebook relies on three separate python files. One (regional_model_scripts) that was written by Angus and myself, and two that I've pinched from GFDL's repos and modified slightly. Perhaps it's easiest for users to import these scripts as needed from the cosima conda package, and then individual notebooks could live in a "MOM6 examples" folder somewhere separate from the source code

### PR DESCRIPTION
**Pull request by navidcy**
Closes #6 !!

### CODE REVIEW

**Comment by navidcy** [comment]
@angus-g, I'm trying to update the installation instructions.

Question: still after merging in conda-forge, to install regional-mom6 we need to **first install `esmpy`** or now `conda install conda-forge::regional-mom6` will deal with everything?

**Comment by angus-g** [comment]
No, the conda method will pull in all the required dependencies.

**Comment by navidcy** [comment]
But the pip method won't, right?

**Comment by angus-g** [comment]
Correct, as noted in the xesmf docs: https://xesmf.readthedocs.io/en/latest/installation.html#notes-about-esmpy

**Comment by navidcy** [comment]
OK, gotcha! Let me update the instructions to reflect this and I'll ping you to review this PR.

**Comment by navidcy** [comment]
have a look whenever you can @angus-g -- thanks!

File: README.md [hunk 1]
```diff
@@ -38,21 +58,25 @@ source](https://earthsystemmodeling.org/esmpy_doc/release/latest/html/install.ht
 With `esmpy` available, we can then install `regional_mom6` via pip. (If we don't have have pip, then
 `conda install pip` should do the job.)
 
-```{code-block} bash
-pip install git+https://github.com/COSIMA/regional-mom6.git
+With `esmpy` install we can now install `regional-mom6` via [`pip`](https://badge.fury.io/py/regional-mom6):
```
Comment (angus-g) [inline README.md:61]:
```suggestion
With `esmpy` installed we can now install `regional-mom6` via [`pip`](https://pypi.org/project/regional-mom6/):
```

File: README.md [hunk 2]
```diff
@@ -24,10 +30,24 @@ Check out the [documentation](https://regional-mom6.readthedocs.io/en/latest/) a
 
 ## Installation
 
-We can install `regional_mom6` via `pip` from GitHub. A prerequisite is the binary `esmpy`
-dependency, which provides regridding capabilities. The easiest way to install `esmpy` is using Conda.
-We encourage using a new or existing conda environment, into which we will install `esmpy` and `regional_mom6`. 
-Then install `emspy` via:
+#### Easy, clean, one liner
```
Comment (angus-g) [inline README.md:33]: This and a couple below, maybe don't skip a header level?

```suggestion
### Easy, clean, one liner
```

File: README.md [hunk 3]
```diff
@@ -24,10 +30,24 @@ Check out the [documentation](https://regional-mom6.readthedocs.io/en/latest/) a
...
+The easiest way is to install `regional-mom6` via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
+We encourage creating a new or using an existing conda environment and then simply
+
+```bash
+conda install conda-forge::regional-mom6
+```
+
+That's it -- now enjoy!
+
+#### "*But I want `pip`, can't I install with `pip`*?"
```
Comment (angus-g) [inline README.md:44]:
```suggestion
### "*But I want `pip`, can't I install with `pip`*?"
```

File: README.md [hunk 4]
```diff
@@ -38,21 +58,25 @@ source](https://earthsystemmodeling.org/esmpy_doc/release/latest/html/install.ht
...
+```bash
+pip install regional-mom6
 ```
 
-The above installs the version of `regional_mom6` (plus any required dependencies) that corresponds
-to the latest commit in GitHub. `esmpy` won't be installed as a dependency and that's why need to
-install it separately.
+The above installs the version of `regional-mom6` (plus any required dependencies) that corresponds to the latest tagged release of the package.
+
+#### "*I'd like to be on the cutting edge of the development*?"
```
Comment (angus-g) [inline README.md:69]:
```suggestion
### "*I'd like to be on the cutting edge of the development*?"
```

File: README.md [hunk 5]
```diff
@@ -38,21 +58,25 @@ source](https://earthsystemmodeling.org/esmpy_doc/release/latest/html/install.ht
...
+First install `esmpy` as described above and then:
 
 ```bash
-pip install git+https://github.com/COSIMA/regional-mom6.git@v0.X.X
+pip install git+https://github.com/COSIMA/regional-mom6.git
 ```
 
-or
+to get the version that corresponds to the latest commit in GitHub.
+Or, install the version that corresponds to a particular git commit via
```
Comment (angus-g) [inline README.md:79]: Maybe make it clearer that this is an example commit?

Also **super** minor nit: you say "via GitHub using" above, but here you say "via", maybe should be "using" here instead?

File: README.md [hunk 6]
```diff
@@ -38,21 +58,25 @@ source](https://earthsystemmodeling.org/esmpy_doc/release/latest/html/install.ht
...
 
-The above installs the version of `regional_mom6` (plus any required dependencies) that corresponds
-to the latest commit in GitHub. `esmpy` won't be installed as a dependency and that's why need to
-install it separately.
+The above installs the version of `regional-mom6` (plus any required dependencies) that corresponds to the latest tagged release of the package.
+
+#### "*I'd like to be on the cutting edge of the development*?"
 
-We can also install `regional_mom6` from a particular tag or git commit using, e.g.,
+Alternatively, we can install directly `regional-mom6` directly via Github using `pip`.
```
Comment (angus-g) [inline README.md:71]:
```suggestion
Alternatively, we can install directly `regional-mom6` directly via GitHub using `pip`.
```

File: docs/installation.md
Comment (angus-g) [inline docs/installation.md:1]: And I guess a reminder to copy any of the suggestions you take in the readme into this file!

**Review by angus-g — COMMENTED** [review]
Thanks! The process is getting cleaner...

File: README.md (hunk 2 above)
Comment (navidcy) [reply README.md:33]: it was just a bit too big with `###` that's why I put 4 `####`

Comment (angus-g) [reply README.md:33]: Fair enough!

File: README.md (hunk 5 above)
Comment (navidcy) [reply README.md]: done

**Comment by navidcy** [comment]
OK, done with everything!

File: docs/installation.md
Comment (navidcy) [reply docs/installation.md:1]: done

**Review by angus-g — APPROVED** [review]
Nice, lgtm

**Pull request merged**